api/scpca_portal/models/computed_file.py

```diff
@@ -246,8 +246,17 @@ def get_sample_multiplexed_file(
         file_name_path_mapping = {}
         for library in libraries:
             library_id = library["scpca_library_id"]
-            for file_suffix in ("_filtered.rds", "_processed.rds", "_qc.html", "_unfiltered.rds"):
-                file_name = f"{library_id}{file_suffix}"
+            file_suffixes = [
+                "filtered.rds",
+                "processed.rds",
+                "qc.html",
+                "unfiltered.rds",
+            ]
+            if not sample.project.includes_cell_lines:
```

Comment:
This should be based off of the sample/library and not the project.

`project.includes_cell_lines` just means that at least one sample/library where `sample.is_cell_line` is true.

The value that we want to base this off of can be found here:
https://github.com/AlexsLemonade/ScPCA-admin/blob/main/scpca-test-data/sample-metadata.tsv

Reply:
Done.